### ISSUE TITLE
Justified image grid list a11y

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#8919]
`<ul>` should be directly followed by `<li>` which isn't currently the case in our justified image grid component. This might require a bit of CSS tweaking to fix, but it's the sort of thing that was flagged in the 2021 a11y report, so good to fix.
